addons/point_of_sale/models/account_payment.py

```diff
@@ -11,14 +11,10 @@ class AccountPayment(models.Model):
     force_outstanding_account_id = fields.Many2one("account.account", "Forced Outstanding Account", check_company=True)
     pos_session_id = fields.Many2one('pos.session', "POS Session")
 
-    def _get_valid_liquidity_accounts(self):
-        result = super()._get_valid_liquidity_accounts()
-        return result + (self.pos_payment_method_id.outstanding_account_id,)
-
-    @api.depends("force_outstanding_account_id")
-    def _compute_outstanding_account_id(self):
+    def _get_outstanding_account_id(self):
         """When force_outstanding_account_id is set, we use it as the outstanding_account_id."""
-        super()._compute_outstanding_account_id()
-        for payment in self:
-            if payment.force_outstanding_account_id:
-                payment.outstanding_account_id = payment.force_outstanding_account_id
+        self.ensure_one()
+        if self.force_outstanding_account_id:
+            return self.force_outstanding_account_id
```

Comment:
wtf is that....

Reply:
Is the issue my adaptation, or the forcing of an outstanding account that already exists?